microsoft-365-ops/3.0/modules/ROOT/pages/microsoft-365-ops-connector-reference.adoc

```diff
@@ -0,0 +1,510 @@
+= Microsoft Dynamics 365 For Operations Connector 3.0 Reference - Mule 4
```

Comment:
So the title is spelled out in different ways in this PR:
Is this Microsoft Dynamics 365 for Finance and Operations Connector, Microsoft Dynamics 365 for Operations Connector, or Microsoft Dynamics 365 for Finance & Operations Connector?